clients/src/main/java/org/apache/kafka/clients/producer/internals/TransactionManager.java

```diff
@@ -1114,7 +1119,7 @@ private boolean maybeTerminateRequestWithError(TxnRequestHandler requestHandler)
         return false;
     }
 
-    private void enqueueRequest(TxnRequestHandler requestHandler) {
+    void enqueueRequest(TxnRequestHandler requestHandler) {
```

Comment:
I think this was made non-private because it had to be visible for testing. Could we add a comment at the top indicating that?

Reply:
Yes. Comment added.